group-coordinator/src/main/java/org/apache/kafka/coordinator/group/GroupCoordinatorConfig.java

```diff
@@ -162,6 +162,18 @@ public class GroupCoordinatorConfig {
     public static final int SHARE_GROUP_MAX_HEARTBEAT_INTERVAL_MS_DEFAULT = 15000;
     public static final String SHARE_GROUP_MAX_HEARTBEAT_INTERVAL_MS_DOC = "The maximum heartbeat interval for share group members.";
 
+    public static final String SHARE_GROUP_RECORD_LOCK_DURATION_MS_CONFIG = "group.share.record.lock.duration.ms";
```

Comment:
we have separate ShareCoordinatorConfig and ShareGroupConfig files
can this be moved to one of them?

Reply:
Indeed. I don't think these record lock configs belong in the GroupCoordinatorConfig. I would say that ShareGroupConfig is best.